Subject: Collected Debian patches for svtplay-dl
Author: Olof Johansson <olof@ethup.se>

The svtplay-dl package is maintained in Git rather than maintaining
patches as separate files, and separating the patches doesn't seem to
be worth the effort.  They are therefore all included in this single
Debian patch.

For full commit history and separated commits, see the packaging Git
repository.
--- svtplay-dl-1.9.1.orig/Makefile
+++ svtplay-dl-1.9.1/Makefile
@@ -4,7 +4,7 @@ all: svtplay-dl
         release clean_releasedir $(RELEASE_DIR)
 
 # These variables describe the latest release:
-VERSION = 1.9.1
+export VERSION = 1.9.1
 LATEST_RELEASE = $(VERSION)
 
 # If we build a new release, this is what it will be called:
--- svtplay-dl-1.9.1.orig/lib/Makefile
+++ svtplay-dl-1.9.1/lib/Makefile
@@ -19,7 +19,9 @@ export PYFILES = $(sort $(addsuffix /*.p
 
 PYTHON ?= /usr/bin/env python
 
-VERSION = $(shell git describe 2>/dev/null || echo $(LATEST_RELEASE)-unknown)
+# This is the string that ends up in the program's --version output
+# Disable this behavior by setting the variable NO_GIT_VERSION=y.
+VERSION$(NO_GIT_VERSION) = $(shell git describe 2>/dev/null || echo $(LATEST_RELEASE)-unknown)
 
 svtplay-dl: $(PYFILES)
 	@# Verify that there's no .build already \
